src/api/firebase/updateFirestore.ts

```diff
@@ -0,0 +1,10 @@
+import { firestore } from './firebase';
+import { doc, updateDoc } from 'firebase/firestore';
+
+export const updateUserFirestore = async (uid: string, data: any) => {
+  await updateDoc(doc(firestore, 'users', uid), data);
+};
+
+export const updateFirestore = async (collection: string, docId: string, data: any) => {
```

Comment:
```suggestion
export const updateFirestoreDoc = async (collection: string, docId: string, data: any) => {
```

Reply:
La functia de updatefield este la fel ca la delete, updateaza una nume field nu toate field-urile pe care le pasam noi in obiect.

Reply:
Practic update-ul general modifica tot obiectul nu doar un field din obiect.